lib/toPairsIn/index.js

```diff
@@ -0,0 +1,10 @@
+export default function toPairsIn(obj) {
+  var values = Object.keys(obj).reduce(function(acc, key) {
+    return acc.concat([[key, obj[key]]])
+  }, [])
```

Comment:
maybe use  `toPairs` here? analyse and check the separate and combined size of changes, is it worth it?